monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandExpirationTimer.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export function CommandExpirationTimer({ minutes, seconds }) {
+
+    function formatDigits(number) {
+      return number.toLocaleString('en-US', { minimumIntegerDigits: 2, useGrouping: false})
+    }
+
+    function generateText() {
+      if (minutes === 0 && seconds === 0) {
+        return 'Command expired';
```

Comment:
Can we regenerate the command and restart the timer here?

Reply:
Note: The `useTimer` method allows one to register an `onExpire` callback

Reply:
https://user-images.githubusercontent.com/11077625/230366820-af7088c3-5143-4ab2-894b-ff6e5d362be9.mov